src/mantarray_waveform_analysis/peak_detection.py

```diff
@@ -106,10 +106,14 @@ def peak_detector(
             if magnetic_signal[valley_idx_last] == magnetic_signal[valley_idx]:
                 valley_indices = np.delete(valley_indices, i)
                 i -= 1
-            else:
-                raise NotImplementedError(
-                    "The only examples observed so far where two peaks were detected were due to the values/heights of the peaks being identical. In that case the decision was made to always choose the earlier peak. This error is a defensive assertion against the future possibility that there might be a case in the future where the values of the two peaks are unequal---in which case we would need to revisit how we would like to handle that outcome."
-                )
+            elif (
+                magnetic_signal[valley_idx] > magnetic_signal[valley_idx_last]
+            ):  # pragma:no-cover
```

Comment:
I checked `.coveragerc` and it looks like the syntax is actually `pragma: no cover`.  not sure if `pragma:no-cover` is working for you or not